src/main/resources/application-local.yml

```diff
@@ -32,3 +32,7 @@ backing-services:
 
 network-store-server:
   base-uri: http://localhost:8080/
+
+# default values for loadflow providers "Hades2" or "OpenLoadFlow"
+loadflow:
+  default-provider: OpenLoadFlow
```

Comment:
should not be set here ?

Reply:
property moved to default application.yml